sourcegraph/sourcegraph
# Add admin onboarding under feature flag

Closes https://github.com/sourcegraph/sourcegraph/issues/43596.

## Test plan
- `sg start`
- Open http://localhost:3080/search page as admin user and check onboarding above search input
- Check individual steps being tracked based on the actions completed on the instance
- Re-sign in and open the same page and check that no onboarding is shown



## Screenshots
| Description | Screenshot |
| --: | :-- |
| In progress | <img width="1728" alt="image" src="https://user-images.githubusercontent.com/6717049/204817816-a602409e-101a-456c-9206-976de272562d.png"> |
| All steps completed | <img width="1728" alt="image" src="https://user-images.githubusercontent.com/6717049/204818083-c828b843-6abd-4a6c-a1cb-bd6e8a1ad227.png"> |
